### ISSUE TITLE
Cannot ingest data into either train_demo or pre-processing

### ISSUE THREAD

**Issue by demongolem-biz** [thunlp/Few-NERD#25]
I finally was able to use Google Colab in conjunction with:

`from datasets import load_dataset
dataset = load_dataset("dfki-nlp/few-nerd", 'supervised')`

to get the supervised data (I am starting with supervised).  It appears train_demo is expecting the two column format on the README page, but the data file train.txt contains 5 columns: index, id, tokens, ner_tags and fine_ner tags.  So that won't work.

So I tried pre-processing.py on the train.txt file, but that tries to load a json file with a column named srcTxt so that won't work.

So in trying to reproduce the supervised results. how do I get the format I have the data in into something that the train_demo file will approve of like is show on the README page.

**Comment by demongolem-biz** [comment]
If we brought `label = torch.cat(query['label'], 0)` before the if block would that be proper?

**Comment by cyl628** [comment]
Hi, please try `bash data/download.sh supervised`. you will get the correct format as two columns. The script `train_demo.py`, however, only contains the implementation of few-shot algorithms, the fully supervised bert-tagger will be added soon.
